docs/guides/summary-guides.md

```diff
@@ -25,14 +25,15 @@ Publisher integrations fall into the following main categories:
 - [Web Integrations](#web-integrations)
 - [Mobile](#mobile)
 - [Google Ad Manager Integrations](#google-ad-manager-integrations)
-- [Prebid Integrations](#prebid-integrations)
 
 ### Web Integrations
 
 The following resources are available for publisher web integrations.
 
 | Integration Guide |  Content Description |
 | :--- | :--- |
+| [Prebid.js Integration Guide](./integration-prebid.md) | An integration guide for publishers who want to integrate with UID2 and generate identity tokens to be passed by Prebid in the RTB bid stream. |
+| [Prebid.js Advanced Integration Guide](./integration-prebid-advanced.md) | An integration guide for publishers who want to integrate with UID2 and generate identity tokens to be passed by Prebid in the RTB bid stream. |
```

Comment:
nit: The same summaries between the two guides make me confused. I think Advanced integration needs server-side changes, could we emphasise this in summary?

Reply:
Done